test/Makefile

```diff
@@ -1,4 +1,4 @@
-DOCKERCOMPOSE := docker-compose -f docker-compose.yml
+DOCKERCOMPOSE := DOCKERGID=`stat -c '%g' /var/run/docker.sock` docker-compose -f docker-compose.yml
```

Comment:
is this needed for all the docker-compose commands or only for grafana?
I mean, can we keep this line clean and add this to the beginning of the line 32?

like this
```
DOCKERCOMPOSE := docker-compose -f docker-compose.yml
...
RUNGRAFANA := DOCKERGID=`stat -c '%g' /var/run/docker.sock` $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSEGRAFANA)
```

Reply:
Done, thank you @tclemos